scripts/lib/api/conversionPipeline.ts

```diff
@@ -78,14 +78,6 @@ export async function runConversionPipeline(
     const { dir, name } = parse(`${markdownPath}/${file}`);
     let url = `/${relative(`${getRoot()}/docs`, dir)}/${name}`;
     results.push({ ...result, url });
-
-    if (
-      pkg.isLatest() &&
-      pkg.hasSeparateReleaseNotes &&
-      file.endsWith("release_notes.html")
-    ) {
-      addNewReleaseNotes(pkg);
-    }
```

Comment:
This wasn't necessary in the for loop. We only need to call it before we call `generateReleaseNotesIndex`. So I moved it to live near that.